vaadin-ordered-layout-flow-demo/src/main/java/com/vaadin/flow/component/orderedlayout/demo/OrderedLayoutView.java

```diff
@@ -699,6 +914,160 @@ private void createVerticalLayoutWithExpandingContent() {
                 layout);
     }
 
+    private void verticalLayoutWithScrolling() {
+        // begin-source-example
+        // source-example-heading: Vertically scrolling
+        VerticalLayout layout = new VerticalLayout();
+        layout.setPadding(true);
+        layout.setHeight("300px");
+
+        Component component1 = createComponent(1, "#78909C");
+        // necessary so that the layout doesn't try to shrink the button.
```

Comment:
This is one of the reasons why the Scroller was built (examples in the same file). Let's not add a demo with hacks for a use-case we already have a built-in component for.

Reply:
Done